libraries/bezier-rs/src/utils.rs

```diff
@@ -29,10 +29,17 @@ pub enum SubpathTValue {
 }
 
 #[derive(Copy, Clone)]
```

Comment:
Could you insert a short comment about this enum's pupose (connects discontinuities in outline / offsets) and then insert a link to the SVG reference? (and do the same for `Cap`)